### ISSUE TITLE
Empty Man pages for Binutils 2.37

### ISSUE THREAD

**Issue by dslm4515** [dslm4515/Musl-LFS#58]
Man pages generated for binutils are empty files. Issue was not known until using pkgtools (makepkg) :

```
WARNING:  zero length file usr/share/man/man1/as.1
WARNING:  zero length file usr/share/man/man1/nm.1
WARNING:  zero length file usr/share/man/man1/ld.1
WARNING:  zero length file usr/share/man/man1/gprof.1
WARNING:  zero length file usr/share/man/man1/objcopy.1
WARNING:  zero length file usr/share/man/man1/strip.1
WARNING:  zero length file usr/share/man/man1/objdump.1
WARNING:  zero length file usr/share/man/man1/addr2line.1
WARNING:  zero length file usr/share/man/man1/strings.1
WARNING:  zero length file usr/share/man/man1/readelf.1
WARNING:  zero length file usr/share/man/man1/c++filt.1
WARNING:  zero length file usr/share/man/man1/elfedit.1
WARNING:  zero length file usr/share/man/man1/ranlib.1
WARNING:  zero length file usr/share/man/man1/size.1
WARNING:  zero length file usr/share/man/man1/ar.1
```
Tested on an older MLFS system with GCC-8.2.0... same issue when building binutild 2.37.

I noticed in `binutils-2.37/binutils/doc` that ar.1 and nm.1 are empty.

**Comment by dslm4515** [comment]
binutils 2.36.0 source does NOT have empty files for those man pages

**Comment by dslm4515** [comment]
git cloning `https://sourceware.org/git/binutils-gdb` does not have this issue of empty man pages

**Comment by dslm4515** [comment]
Fixed in commit 997317cf5c0ad78df7f33fdae3168f75487d1041
